apps/docs/components/ui/table/TypographyTable.tsx

```diff
@@ -0,0 +1,171 @@
+"use client";
+
+import React from "react";
+import TypographyPreview from "@/components/preview/TypographyPreview";
+import Code from "@/components/ui/code/Code";
+
+import "./table.css";
+
+interface TypographyTableProps {
+    type: string;
+    data: Record<string, { name: string; value: string }[]>;
+}
+
+type FontProperty = "fontFamily" | "fontSize" | "fontWeight" | "lineHeight";
+
+type Size = "3xl" | "2xl" | "xl" | "lg" | "md" | "sm" | "xs";
+
+type GroupedItems = Record<Size, Record<FontProperty, string>>;
+
+const TypographyTable = ({ type, data }: TypographyTableProps) => {
+    interface TokenData {
```

Comment:
This was already said in the previous review, but still not modified. interface TokenData needs to be outside the component